specification/columns/hostprovider.md

```diff
@@ -0,0 +1,38 @@
+# Host Provider
+
+A Host Provider is an entity whose [*resources*](#glossary:resource) are used by the Provider to make their [*resources*](#glossary:resource) or [*services*](#glossary:service) available.
+
+The HostProvider column adheres to the following requirements:
+
+* The HostProvider column MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) and represents the service provider where [*resources*](#glossary:resource) or [*services*](#glossary:service) are deployed on.
+* For Providers that allow their customer to select or know the host provider where resources or services are deployed, this SHOULD be the name of that selected or visible host provider
```

Comment:
nit: Do we use the term "customer"? I thought we use "practitioner"? Also applies on other lines.